packages/core/admin/admin/src/utils/unique-admin-hash.js

```diff
@@ -0,0 +1,13 @@
+const hash = require('hash.js');
```

Comment:
Not asking you to change but just wanted to note it somewhere, [Subtle Crypto](https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto) is available in the browser, they do provide a `digest` function.

The only issue I can see with that is that it's only available in a secure context (HTTPS), which could be painful for our use case. Let's keep that in mind to reduce the bundle size.

Reply:
Thanks for investigating that. I'll keep that in mind and we'll be able to swap it out if the solution works for our use case 😊👍